src/service/chat.js

```diff
@@ -114,15 +114,15 @@ const sendAnonymousDMOtherProfile = async ({
   message
 }) => {
   const payload = {
-    members: [user_id],
+    member: user_id,
     message,
     anon_user_info_emoji_name,
     anon_user_info_emoji_code,
     anon_user_info_color_name,
     anon_user_info_color_code
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The payload for the API request has been changed from an array of members to a single member. This change could potentially break the functionality if the backend service is expecting an array of members. Please ensure that the backend service is updated accordingly to handle this change.
```diff
-    members: [user_id],
+    member: user_id,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->